Documentation/Storage-Configuration/node-maintenance.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Let's move this to the "Troubleshooting" section

Reply:
okay sure